DNN Platform/Library/Entities/Tabs/TabController.cs

```diff
@@ -1469,6 +1469,33 @@ public TabCollection GetTabsByPortal(int portalId)
                                                             });
         }
 
+        /// <summary>
+        /// Get the actual visible tabs for a given portal id. 
+        /// System Tabs and Admin Tabs are excluded from the result set.
+        /// </summary>
+        /// <param name="portalId"></param>
+        /// <param name="adminTabId"></param>
+        /// <returns></returns>
+        public TabCollection GetTabsByPortal(int portalId, int adminTabId)

```

Comment:
Shouldn't we adapt the old code using optional parameters? Having definition as `public TabCollection GetTabsByPortal(int portalId, int? adminTabId = null)` and then have an `if` statement before returning to decide if it needs to be filtered or not? This way we avoid code duplication.